label_studio/webhooks/admin.py

```diff
@@ -1,3 +1,2 @@
-from django.contrib import admin
```

Comment:
I'm in favor of deleting this empty admin module as well, as long as doing so doesn't cause issues in the admin.